_pages/home.md

```diff
@@ -10,6 +10,9 @@ excerpt: "Special Interest Group for Building Educational Applications<br/><br/>
 
 <h2>News</h2>
 
+**August 5, 2019**. The proceedings of the 14th Workshop on Building Educational Applications are now available [online](https://www.aclweb.org/anthology/W19-44)!
```

Comment:
@ekochmar I think it might be better to link to the HTML version of the proceedings: https://www.aclweb.org/anthology/events/ws-2019/#w19-44 since that would load much faster. And folks can get the PDF from there if they really need it.